src/Tvheadend.cpp

```diff
@@ -664,7 +664,8 @@ PVR_ERROR CTvheadend::GetTimerTypes ( PVR_TIMER_TYPE types[], int *size )
   }
 
   /* PVR_Timer.iPreventDuplicateEpisodes values and presentation.*/
-  static std::vector< std::pair<int, std::string> > deDupValues;
+  std::vector< std::pair<int, std::string> > deDupValues;
```

Comment:
Damn, somehow I didn't experienced this while testing. 
@ksooo Thanks for the fix! And indeed I was wrong :-)